[2023-12-30T13:03:01.283+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.sense_json_file scheduled__2023-12-24T00:00:00+00:00 [queued]>
[2023-12-30T13:03:01.583+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.sense_json_file scheduled__2023-12-24T00:00:00+00:00 [queued]>
[2023-12-30T13:03:01.590+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-30T13:03:02.533+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): sense_json_file> on 2023-12-24 00:00:00+00:00
[2023-12-30T13:03:02.860+0000] {standard_task_runner.py:57} INFO - Started process 313 to run task
[2023-12-30T13:03:03.163+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'sense_json_file', 'scheduled__2023-12-24T00:00:00+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/dag0001.py', '--cfg-path', '/tmp/tmpkk7w_fbi']
[2023-12-30T13:03:03.346+0000] {standard_task_runner.py:85} INFO - Job 169: Subtask sense_json_file
[2023-12-30T13:03:04.122+0000] {task_command.py:416} INFO - Running <TaskInstance: my_dag.sense_json_file scheduled__2023-12-24T00:00:00+00:00 [running]> on host e1038f319166
[2023-12-30T13:03:04.630+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Javad' AIRFLOW_CTX_DAG_ID='my_dag' AIRFLOW_CTX_TASK_ID='sense_json_file' AIRFLOW_CTX_EXECUTION_DATE='2023-12-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-24T00:00:00+00:00'
[2023-12-30T13:03:04.638+0000] {logging_mixin.py:154} INFO - Sensing JSON file
[2023-12-30T13:03:04.645+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-30T13:03:04.673+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=sense_json_file, execution_date=20231224T000000, start_date=20231230T130301, end_date=20231230T130304
[2023-12-30T13:03:04.857+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-30T13:03:05.031+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
